client/src/services/bms/types.ts

```diff
@@ -60,6 +63,18 @@ interface PersonalInformationResultData {
   job?: string;
 }
 
+interface UpdatePersonalInformationResultData {
```

Comment:
I don't think you need empty interfaces if there's no data to return. Just updating the enum is enough.

Reply:
done